box.json.dist

```diff
@@ -1,23 +1,16 @@
 {
-    "alias": "psalm.phar",
-    "base-path": "build/psalm",
-    "main" : "./psalm",
-    "output" : "../../build/psalm.phar",
-    "directories" : [
-        "assets",
-        "vendor"
-    ],
-    "finder": [
-        {
-            "in" : "src"
-        }
-    ],
-    "files" : [
-        "config.xsd"
+    "output" : "build/psalm.phar",
+    "files": [
+        "src/command_functions.php",
+        "src/psalm.php",
+        "src/psalter.php"
     ],
+    "files-bin": ["config.xsd"],
+    "directories-bin" : ["assets"],
     "intercept" : false,
-    "compactors" : [],
+    "compactors" : [
+        "KevinGH\\Box\\Compactor\\PhpScoper"
+    ],
     "chmod" : "0755",
-    "shebang" : "#!/usr/bin/env php",
-    "stub" : true
+    "compression": "GZ"
```

Comment:
there was no compression, not sure if that was voluntary because you wanted to avoid your users to have the `zlib` extension

Reply:
Yes, definitely on purpose (because `bzip` was causing issues for users without it). If 99% of PHP installs have `zlib`, happy to use that.